src/main/java/org/gatorgradle/task/GatorGradleTask.java

```diff
@@ -35,10 +35,10 @@
 
 public class GatorGradleTask extends DefaultTask {
   // The executor to use to execute the grading
-  private final WorkerExecutor executor;
+  protected final WorkerExecutor executor;
```

Comment:
This should only be in the grade task, as it shouldn't be needed for just reporting.